NebulaHost/PacketProcessors/Logistics/StationUIBroadcaster.cs

```diff
@@ -22,18 +22,37 @@ public StationUIBroadcaster()
         }
         public void ProcessPacket(StationUI packet, NebulaConnection conn)
         {
-            Player player = playerManager.GetPlayer(conn);
             // if a user adds/removes a ship, drone or warper or changes max power input broadcast to everyone.
-            if (player != null && StationUIManager.UpdateCooldown == 0 &&
+            if (StationUIManager.UpdateCooldown == 0 &&
                 (packet.SettingIndex == StationUI.EUISettings.MaxChargePower
                  || packet.SettingIndex == StationUI.EUISettings.SetDroneCount
                  || packet.SettingIndex == StationUI.EUISettings.SetShipCount
                  || packet.SettingIndex == StationUI.EUISettings.SetWarperCount
-                 || packet.SettingIndex == StationUI.EUISettings.AddOrRemoveItemFromStorageRequest
-                 || packet.SettingIndex == StationUI.EUISettings.AddOrRemoveItemFromStorageResponse)
+                 )
```

Comment:
Don't like the lone bracket here, looks better on the line above as was before

Reply:
Code style rule on this should be enforced. The advantage of this format is that you don't add extra diff noise by adding new conditions.

Reply:
Adding them back produces
```diff
                 || packet.SettingIndex == StationUI.EUISettings.SetWarperCount
+                || packet.SettingIndex == StationUI.EUISettings.AddOrRemoveItemFromStorageRequest
+                || packet.SettingIndex == StationUI.EUISettings.AddOrRemoveItemFromStorageResponse
				)